code/game/mecha/combat/marauder.dm

```diff
@@ -145,3 +145,17 @@
 						"}
 	output += ..()
 	return output
+
+//Meant for random spawns.
+/obj/mecha/combat/marauder/old
+	desc = "Heavy-duty, combat exosuit, developed after the Durand model. Rarely found among civilian populations. This one is particularly worn looking and likely isn't as sturdy."
+
+/obj/mecha/combat/marauder/old/New()
+	..()
+	health = 25
+	maxhealth = 400	//Just slightly worse.
+	cell.charge = rand(0, (cell.charge/2))
+
+/obj/mecha/combat/marauder/old/Initialize()	//Nothing for u

```

Comment:
This won't actually stop it from spawning the equipment, as it's just calling the base marauder's Initialize.

Reply:
Fixed in commit.